table/column.go

```diff
@@ -191,6 +192,11 @@ func CastValue(ctx sessionctx.Context, val types.Datum, col *model.ColumnInfo, r
 		truncateTrailingSpaces(&casted)
 	}
 
+	if col.Tp == mysql.TypeFloat && casted.GetFloat32() > math.MaxFloat32 {
```

Comment:
Why don't we move this logic to function `ConvertTo`?

Reply:
Thx.